services/common-controls.yaml

```diff
@@ -117,14 +165,34 @@ controls:
       NIST_800_53:
         - AC-3
     test_requirements:
-      tlp_red:
-        01: |
-          Verify that endpoints are blocked from public access, except for those explicitly added to an allowlist.
-        02: |
-          Confirm that services deployed within the same cloud tenant are blocked by default and only accessible to allowlisted services.
-        03: |
-          Ensure access from other cloud tenants is blocked by default, and only entities on the allowlist can access the service.
-  - id: CCC.C06
+      - id: CCC.C05.TR01
+        text: |
+          The service blocks access to sensitive resources and admin access
+          from untrusted sources, including unauthorized IP addresses, domains,
+          or networks that are not included in a pre-approved allowlist.
+        tlp_levels:
+          - tlp_clear
+          - tlp_green
+          - tlp_amber
+          - tlp_red
+      - id: CCC.C05.TR02
+        text: |
+          The service logs all access attempts from untrusted entities,
+          including failed connection attempts.
+        tlp_levels:
+          - tlp_amber
+          - tlp_red
+      - id: CCC.C05.TR04
+        text: |
+          The service prevents unauthorized cross-tenant access, ensuring that
+          only allowlisted services from other tenants can access resources.
```

Comment:
what do we mean by "tenant" here? I think that's worth expanding on.

Reply:
This change isn't in the current PR after merging 418, but I think it's worth spinning back around to if you'd like to propose a change to common controls